serverless/src/registration.js

```diff
@@ -1,20 +1,50 @@
 const AWS = require("aws-sdk");
-
+const { httpResponse } = require("./http_response");
 exports.handler = async (event) => {
-  const dbClient = new AWS.DynamoDB.DocumentClient();
-
-  exports.handler = async (event) => {
-    let body;
-    let statusCode = 200;
-    try {
-      body = await dbClient.put(event).promise();
-    } catch (err) {
-      statusCode = err.statusCode;
-      body = err.message;
-    }
-    return {
-      statusCode,
-      body,
-    };
+  const docClient = new AWS.DynamoDB.DocumentClient();
+  let body;
+  let statusCode = 201;
+  let headers = {
+    "Content-Type": "application/json",
```

Comment:
Can you add these headers directly in the httpResponse.js file. That way every request will be able to use them, in case we have other post requests int the future. It should be fine to include them even if they are not used for some requests